lidarrmetadata/api.py

```diff
@@ -128,7 +128,7 @@ def get_artist_info(mbid):
     return jsonify(artist)
 
 
-@app.route('/album/<mbid>/', methods=['GET'])
+@app.route('/album/<mbid>', methods=['GET'])
```

Comment:
This causes issues in Lidarr, when removing from the Lidarr side search fails. I don't think these were causing the issues.